app/assets/javascripts/order.js

```diff
@@ -0,0 +1,39 @@
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (2% scanned).